pkg/sessionctx/vardef/tidb_vars.go

```diff
@@ -1676,6 +1677,7 @@ var (
 	PasswordHistory                 = atomic.NewInt64(DefPasswordReuseHistory)
 	PasswordReuseInterval           = atomic.NewInt64(DefPasswordReuseTime)
 	IsSandBoxModeEnabled            = atomic.NewBool(false)
+	MaxUserConnectionsCount         = atomic.NewUint32(DefMaxUserConnections)
```

Comment:
```suggestion
	MaxUserConnectionCount         = atomic.NewUint32(DefMaxUserConnections)
```

Reply:
> 

Remove `MaxUserConnectionsCount` to `MaxUserConnectionsValue`.